remo/events/static/events/js/events_list.js

```diff
@@ -44,8 +44,9 @@ var ATTENDANCE_LEVEL = {
 
 function initialize_map() {
     // Initialize map.
+    L.Icon.Default.imagePath = '/static/base/css/images/';
```

Comment:
is there a way we don't need to set that at every use?

Reply:
We need that because the js library is calculating the path of image files using the path of the js files and we have in a complete different path.

Reply:
But can't we do that globally?